lms/djangoapps/instructor/views/api.py

```diff
@@ -2846,7 +2846,13 @@ def parse_request_data_and_get_user(request):
     try:
         db_user = get_user_by_username_or_email(user)
     except ObjectDoesNotExist:
-        raise ValueError(_('Student (username/email={user}) does not exist').format(user=user))
+        raise ValueError(_("We can't find the user you've entered. Make sure the username or email address is correct,"
+                           " then try again."))
+
+    # Make Sure the given student is enrolled in the course
+    if not CourseEnrollment.is_enrolled(db_user, course_key):
+        raise ValueError(_("The user you have entered is not enrolled in this course. "
```

Comment:
Here as well I would include the username/email for the user in the error message